controller: {
    KP_x: 0.3,
    KI_x: 0.05,
    KD_x: 0.1,
    X_limit: 0.2,
    
    KP_y: 0.3,
    KI_y: 0.05,
    KD_y: 0.1,
    Y_limit: 0.2,
    
    KP_z: 0.18,
    KI_z: 0.05,
    KD_z: 0.08,
    Z_limit: 0.2,
    
    KP_yaw: 0.1,
    KI_yaw: 0.05,
    KD_yaw: 0.05,
    
    TAKEOFF_HEIGHT: 0.05,
    TAKEOFF_THROTTLE: 0.3,
    TAKEOFF_THROTTLE_TIME: 0.6,
    LAND_TIME: 5.0,
    
    Yaw: 0,
    Flip: 100
}
